static/js/src/entry/business/Choices.vue

```diff
@@ -1,18 +1,16 @@
 <template>
```

Comment:
Nitpicky, but should we remain the two `Choices.vue` files `FormBucket.vue` or something like that so they line up more neatly with the class names?

Reply:
for sure, always a fan of matching up like that. I'll get that updated. Thanks!